src/pages/Trends/connectedWordsEpic.js

```diff
@@ -0,0 +1,234 @@
+import { Observable } from 'rxjs'
+import { handleErrorAndTriggerAction } from '../../epics/utils'
+import {
+  TRENDS_HYPED_FETCH_SUCCESS,
+  TRENDS_HYPED_FETCH_TICKERS_SLUGS_SUCCESS,
+  TRENDS_CONNECTED_WORDS_SUCCESS,
+  TRENDS_CONNECTED_WORDS_FAILED,
+  TRENDS_CONNECTED_WORDS_OPTIMIZATION_SUCCESS
+} from '../../components/Trends/actions'
+import { ALL_INSIGHTS_BY_TAG_QUERY } from '../../components/Insight/insightsGQL'
+import { binarySearch } from './utils'
+
+const oneDayTimestamp = 1000 * 60 * 60 * 24
+
+const getInsightTrendTagByDate = date =>
+  `${date.getDate()}-${date.getMonth()}-${date.getFullYear()}-trending-words`
+
+const tickerCheckFn = (target, { ticker }) => target === ticker
```

Comment:
Why did you use Fn in names? @DmitriVanGuard

Reply:
Really was not thinking about it when created. Maybe better to call it with `Clb` at the end?